src/Ionide.ProjInfo/InspectSln.fs

```diff
@@ -97,9 +97,16 @@ module InspectSln =
                             not (isNull item.Parent)
                             && item.Parent.Id = folder.Id
                         )
-                        |> Seq.map (fun p -> parseItem p, string p.Id)
-                        |> List.ofSeq
-                        |> List.unzip
+                        |> Seq.map (fun p -> parseItem p)
+                        |> List.ofSeq,
+
+                        folder.Files
```

Comment:
folder.Files can be null here, hence the Option.ofObj stuff to turn such a case into an empty collection